tests/apollo/util/apollo_logging.py

```diff
@@ -0,0 +1,36 @@
+from eliot import start_action, start_task, to_file, add_destinations
+from datetime import datetime
+import os
+
+
+def set_file_destination():
+    test_name = os.environ.get('TEST_NAME')
```

Comment:
instead of using an environment variable, can you try with:
```python
test_name = sys.modules['__main__'].__file__
```
?

Reply:
Awesome!